src/model/build-parameters.ts

```diff
@@ -190,7 +192,7 @@ class BuildParameters {
       branch: Input.branch.replace('/head', '') || (await GitRepoReader.GetBranch()),
       cloudRunnerBranch: CloudRunnerOptions.cloudRunnerBranch.split('/').reverse()[0],
       cloudRunnerDebug: CloudRunnerOptions.cloudRunnerDebug,
-      githubRepo: Input.githubRepo || (await GitRepoReader.GetRemote()) || 'game-ci/unity-builder',
+      githubRepo: (Input.githubRepo ?? (await GitRepoReader.GetRemote())) || 'game-ci/unity-builder',
```

Comment:
Why the change to nullish coalesce? Should it not be fixed if that would ever throw?

Reply:
Can certainly revert it. Was just a suggestion from sonarlint that I was accepting throughout the file.